packages/flutter/lib/src/painting/flutter_logo.dart

```diff
@@ -47,11 +47,16 @@ class FlutterLogoDecoration extends Decoration {
     this.textColor: const Color(0xFF616161),
     this.style: FlutterLogoStyle.markOnly,
     this.margin: EdgeInsets.zero,
-  }) : _position = style == FlutterLogoStyle.markOnly ? 0.0 : style == FlutterLogoStyle.horizontal ? 1.0 : -1.0, // ignore: CONST_EVAL_TYPE_BOOL_NUM_STRING
+  }) : assert(lightColor != null),
+       assert(darkColor != null),
+       assert(textColor != null),
+       assert(style != null),
+       assert(margin != null),
```

Comment:
Should the comment above mention that margin also cannot be null?

Reply:
Yes! Good catch. Fixed. Thanks.